stable/restore/values.yaml

```diff
@@ -32,7 +32,7 @@ nuodb:
   image:
     registry: docker.io
     repository: nuodb/nuodb-ce
```

Comment:
One day - when we have a `client-only` container image, we can use that here - since `nuorestore` only uses `nuocmd`.